packages/next/src/executors/build/build.impl.ts

```diff
@@ -49,32 +49,20 @@ export default async function buildExecutor(
     process.env['__NEXT_REACT_ROOT'] ||= 'true';
   }
 
-  const { experimentalAppOnly, profile, debug, outputPath } = options;
-
   // Set output path here since it can also be set via CLI
   // We can retrieve it inside plugins/with-nx
-  process.env.NX_NEXT_OUTPUT_PATH ??= outputPath;
+  process.env.NX_NEXT_OUTPUT_PATH ??= options.outputPath;
```

Comment:
Why not set this also in the `runCliBuild` since we already extract the `outputPath` there?

Reply:
I could it has nothing to do with the CLI build itself though